src/input-range-slider/input-range-slider.styles.ts

```diff
@@ -29,6 +29,10 @@ export const LabelContainer = styled.div`
     gap: 1rem;
 `;
 
+export const IndicationLabelContainer = styled.div`
+    margin-bottom: 0.25rem;
```

Comment:
should be 1 rem based on figma

Reply:
Based on the figma, the distance between the indicator and the label below should be the same, think the track got height affected, and the label also using margin-top:0.25rem;  to achieve the visible 1rem

Reply:
Solved, updated both label (bottom mix and max, and indicator margin to 1rem)